.circleci/config.yml

```diff
@@ -105,10 +105,46 @@ jobs:
       - store_artifacts:
           path: test-reports
 
+  # oldest supported dependencies build
+  oldest_supported:
+    docker:
+      - image: continuumio/miniconda:latest
+    environment:
+      PYTHON: "3.6"
+      ENV_NAME: "dask-ml-test"
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps-{{ .Branch }}-oldest_supported-{{ checksum "ci/environment-3.6.yml" }}
+      - run:
+          name: configure conda
+          command: ./ci/install-circle.sh
+      - run:
+          name: install oldest dependencies
+          command: |
+            source activate ${ENV_NAME}
+            pip uninstall -y dask distributed
+            conda uninstall -y --force pandas scikit-learn multipledispatch numpy numpy-base
```

Comment:
I had to uninstall `numpy` and `numpy-base` here even though it's unrelated to the oldest supported versions of dask, pandas, etc. Based on some local testing, my best guess is that this is because we're installing packages from both the `defaults` and `conda-forge` channels in `ci/environment-3.6.yml`. Any objections to using just the `conda-forge` channel?

Reply:
No objection.